### PR TITLE
Fix category path when placed on root

### PR DESCRIPTION
**Pull request by giuseppelt**
This patch fixes the double slashes (i.e. http://site.com//cat1) in the category path, when you configure `category_dir` with '/' or ''

### CODE REVIEW

File: lib/models/category.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ module.exports = function(ctx) {
 
   Category.virtual('path').get(function() {
     var catDir = ctx.config.category_dir;
-    if (catDir[catDir.length - 1] !== '/') catDir += '/';
+    if (catDir === '/') catDir = '';
+    if (catDir.length > 0 && catDir[catDir.length - 1] !== '/') catDir += '/';
```
Comment (tommy351) [inline lib/models/category.js]: `catDir.length > 0` can be `catDir.length`.

Comment (giuseppelt) [inline lib/models/category.js]: what you mean? I know the evaluation is the same, I used explicit comparison for better reading.
Have I remove `> 0`? If Y, no prob.

Comment (Xuanwo) [inline lib/models/category.js]: Look like to keep code style?

**Comment by kbaley** [comment]
Wondering when this will be merged. We're looking to go live with a site we've converted from Jekyll and we need this to match our existing URLs.

**Comment by leesei** [comment]
Sorry we are a bit slow in merging the PR.
Please use `npm install` your own repo for your release.

**Comment by leesei** [comment]
Merged via https://github.com/hexojs/hexo/commit/9aca27b2bddaca35d22880d66aa03d77c6114a4e.
Thanks.

**Pull request closed without merging**